.travis.yml

```diff
@@ -23,10 +26,14 @@ addons:
 before_script:
 - . ./test/prism.sh
 - prism version
+- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+- chmod +x ./cc-test-reporter
+- ./cc-test-reporter before-build
 script:
 - if [[ $TRAVIS_PYTHON_VERSION == '2.6' ]]; then coverage run -m unittest2 discover; else coverage run -m unittest discover; fi
 after_script:
 - codecov
+- ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
```

Comment:
@prashuchaudhary it looks like there's an issue:
```
$ ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
ERRO[0000] failed to read file __init__.py
open __init__.py: no such file or directory 
Error: open __init__.py: no such file or directory
Usage:
  cc-test-reporter after-build [flags]
Flags:
  -s, --batch-size int               batch size for source files (default 500)
  -e, --coverage-endpoint string     endpoint to upload coverage information to (default "https://api.codeclimate.com/v1/test_reports")
  -t, --coverage-input-type string   type of input source to use [simplecov, lcov, coverage.py, clover, gocov, gcov, cobertura, jacoco]
      --exit-code int                exit code of the test run
  -r, --id string                    reporter identifier
  -p, --prefix string                the root directory where the coverage analysis was performed (default "/home/travis/build/sendgrid/sendgrid-python")
Global Flags:
  -d, --debug   run in debug mode
```

Reply:
will check and update

Reply:
Thank you!

Reply:
@prashuchaudhary please let me know when you get a chance to look at this and I will loop back around.